.github/workflows/wheels.yml

```diff
@@ -0,0 +1,57 @@
+name: Build Python Bindings
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  build-sdist:
+    name: Build sdist
+    runs-on: ubuntu-latest
+    steps:
+      - name:   Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name:   Install build dependencies
+        run: |
+          python -m pip install build
+
+      - name: 📦 Build the sdist
+        working-directory: ./openvdb/openvdb/python
+        run: |
+          python -m build --sdist
+
+      - name:   Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: ./openvdb/openvdb/python/dist/
+
+  build-wheels:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - name:   Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: 🛞 Build wheels
+        uses: pypa/cibuildwheel@v2.13.1
+        with:
+          package-dir: ./openvdb/openvdb/python
+          config-file: ./openvdb/openvdb/python/pyproject.toml
+        env:
+          CIBW_ARCHS_MACOS: auto universal2
```

Comment:
I think you should set the manylinux versions to make sure you target CentOS 7.

Reply:
What do you think about targeting something `manylinux_2_28`? I'd rather not target CentOS 7 because although in many projects it is still the default build platform, it's discontinued, and basically the entire reason this PR didn't get merged with wheels before was because it uses an old build toolchain that OpenVDB didn't support, and there was no reasonable way to update it.

Reply:
28 is way too recent. The manylinux_2014 container I believe has a newer compiler with the CentOS 7 glibc. It's all about glibc.

Reply:
Sure, we can use that - it's the default for cibuildwheel currently, but setting the appropriate options in `pyproject.toml` will ensure it stays that way.